### PR TITLE
Improve r2dii.plot article

### PR DESCRIPTION
**Pull request by MonikaFu**
In this PR I improve article text so that it is more clear and easier to follow.

Happy to see feedback on the text itself @maurolepore !

### CODE REVIEW

**Review by maurolepore — APPROVED** [review]
LGTM. I really like your edits! I made very few comments.

Also, something not in this PR:

I would call `library(r2dii.plot)` last -- so as to highlight it and ensure its functions overwrite any other function from other pacakages attached above.

Good

```r
library(pkg1)
library(pkg2)
library(most.important.package)
```

Not so good

```r
library(pkg1)
library(most.important.package)
library(pkg2)
```

File: vignettes/r2dii-plot.Rmd [hunk 1]
```diff
@@ -135,20 +150,27 @@ plot_emission_intensity(data, convert_label = to_title, span_5yr = TRUE) +
   )
 ```
 
-You can polish your plot by modifying the input `data` and output "ggplot"
-object. Your options are limitless but these are some typical things you may do:
+### Plot customization strategies 3 and 2: input data and ggplot2 functions
```
[the remark this answers is not in the document]
Comment (MonikaFu) [reply vignettes/r2dii-plot.Rmd]: Thanks! Good point, I re-ordered them.

File: vignettes/r2dii-plot.Rmd [hunk 2]
```diff
@@ -105,11 +112,19 @@ qplot_techmix(data)
...
+3) Modify the input `data`. 
+
+The following sub sections show the three strategies applied to different functions 
+of the `plot_*()` family.
 
-The basic output of `plot_emission_intensity()` looks rather unappealing.
+### Plot customization strategies 1 and 2: parameters and ggplot2 functions
+
+We will show how you might customize a plot using strategy 3 and 2 based on `plot_emission_intensity()`. The basic output of the function looks rather 
+unappealing.
```
[the remark this answers is not in the document]
Comment (MonikaFu) [reply vignettes/r2dii-plot.Rmd]: Right, it is a typo.

**Comment by MonikaFu** [comment]
Thanks @maurolepore for your comments, I addressed them and merging now.

**Pull request merged**